### ISSUE TITLE
[Bug] Crash on start up

### ISSUE THREAD

**Issue by gab003** [BiscuitDevelopment/SkyblockAddons#503]
Please note that we do not provide support for clients other than Forge.
We will not fix compatibility issues with other clients.

**Describe the bug**
Crashes on startup.

**To Reproduce**
Build beta 7 from development branch

**Expected behavior**
Should have launched

**Crash Report**
If your game crashed, please paste your crash report below.

```text
WARNING: coremods are present:
  FMLLoadingPlugin (Scrollable Tooltips-1.3 (1.8.9).jar)
  SkyblockAddonsLoadingPlugin (SkyblockAddons-1.6.0-beta.7-for-MC-1.8.9.jar)
  PatcherTweaker (Patcher-1.5.1 (1.8.9).jar)
  ModTweaker (Patcher-1.5.1 (1.8.9).jar)
Contact their authors BEFORE contacting forge

// This doesn't make any sense!

Time: 14/02/21 11:09
Description: There was a severe problem during mod loading that has caused the game to fail

net.minecraftforge.fml.common.LoaderException: java.lang.NoSuchMethodError: com.google.gson.JsonParser.parseString(Ljava/lang/String;)Lcom/google/gson/JsonElement;
	at net.minecraftforge.fml.common.LoadController.transition(LoadController.java:162)
	at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:744)
	at net.minecraftforge.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:310)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:495)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:329)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.NoSuchMethodError: com.google.gson.JsonParser.parseString(Ljava/lang/String;)Lcom/google/gson/JsonElement;
	at codes.biscuit.skyblockaddons.utils.Utils.loadLanguageFile(Utils.java:662)
	at codes.biscuit.skyblockaddons.utils.Utils.loadLanguageFile(Utils.java:645)
	at codes.biscuit.skyblockaddons.config.ConfigValues.loadValues(ConfigValues.java:243)
	at codes.biscuit.skyblockaddons.SkyblockAddons.postInit(SkyblockAddons.java:140)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:560)
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:211)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:189)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:118)
	at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:742)
	... 10 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.8.9
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 276553000 bytes (263 MB) / 1260912640 bytes (1202 MB) up to 3817865216 bytes (3641 MB)
	JVM Flags: 2 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx4G
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.19 Powered by Forge 11.15.1.1722 Optifine OptiFine_1.8.9_HD_U_M5 13 mods loaded, 13 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJ	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJ	FML{8.0.99.99} [Forge Mod Loader] (forge-1.8.9-11.15.1.1722.jar) 
	UCHIJ	Forge{11.15.1.1722} [Minecraft Forge] (forge-1.8.9-11.15.1.1722.jar) 
	UCHIJ	autotip{3.0.1} [Autotip] (Autotip-3.0.1 [1.8-1.12.2].jar) 
	UCHIJ	modcore{0.1.47} [ModCore] (Sk1er Modcore-0.1.47 (1.8.9).jar) 
	UCHIJ	Danker's Skyblock Mod{1.8.4} [Danker's Skyblock Mod] (1.8.9.Danker.s.Skyblock.Mod.-.1.8.4.jar) 
	UCHIJ	sidebarmod{2.0} [Sidebar Mod Revamp] ([1.8.9] Sidebar Mod Revamp.jar) 
	UCHIJ	konkrete{1.1.3} [Konkrete] (konkrete_1.1.3_MC_1.8.9.jar) 
	UCHIJ	fancymenu{1.6.0.1} [FancyMenu] (fancymenu_1.6.0.1_MC_1.8.9.jar) 
	UCHIJ	patcher{1.5.1} [Patcher] (Patcher-1.5.1 (1.8.9).jar) 
	UCHIJ	scamprotect{1.2.6} [ScamProtect for 1.8.9] (scamprotect-1.8.9-1.2.6.jar) 
	UCHIJ	text_overflow_scroll{1.3} [Scrollable Tooltips] (Scrollable Tooltips-1.3 (1.8.9).jar) 
	UCHIE	skyblockaddons{1.6.0-beta.7} [SkyblockAddons] (SkyblockAddons-1.6.0-beta.7-for-MC-1.8.9.jar) 
	Loaded coremods (and transformers): 
FMLLoadingPlugin (Scrollable Tooltips-1.3 (1.8.9).jar)
  club.sk1er.mods.overflowscroll.forge.ClassTransformer
SkyblockAddonsLoadingPlugin (SkyblockAddons-1.6.0-beta.7-for-MC-1.8.9.jar)
  codes.biscuit.skyblockaddons.tweaker.SkyblockAddonsTransformer
PatcherTweaker (Patcher-1.5.1 (1.8.9).jar)
  club.sk1er.mods.core.forge.ClassTransformer
  club.sk1er.patcher.tweaker.ClassTransformer
ModTweaker (Patcher-1.5.1 (1.8.9).jar)
  club.sk1er.patcher.tweaker.other.ModClassTransformer
	GL info: ' Vendor: 'ATI Technologies Inc.' Version: '4.6.14757 Compatibility Profile Context 20.11.2 27.20.14501.18003' Renderer: 'AMD Radeon RX 5700 XT'
	OptiFine Version: OptiFine_1.8.9_HD_U_M5
	OptiFine Build: 20210124-163719
	Render Distance Chunks: 8
	Mipmaps: 4
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 4.6.14757 Compatibility Profile Context 20.11.2 27.20.14501.18003
	OpenGlRenderer: AMD Radeon RX 5700 XT
	OpenGlVendor: ATI Technologies Inc.
	CpuCount: 12
```

**System Information**
 - OS: Windows 10
 - Minecraft Version: 1.8.9
 - Forge Version: Both 11.15.1.2318 & 11.15.1.1722
 - SkyblockAddons Version: Beta 7

**Other Installed Mods**
I have tried to launch it without the mods I have installed but the same problem occours

**Comment by PumpkinXD** [comment]
Duplicate of #467

**Comment by gab003** [comment]
> 
> 
> Duplicate of #467
> fix : #469
> ( biscuit said he fixed this bug locally, but development branch haven't fix it so you need fix it manually)

Ah, cheers for fix.
